app/controllers/index.js

```diff
@@ -17,22 +17,24 @@ export default class IndexController extends Controller {
     let users = await all(fetchRequests);
     users = this.identifyUsers(users);
     users = this.sortUsers(users);
+    users = await this.fetchAdditionalUserDetails(users);
 
     this.updateContributorsList(users);
   }
 
   identifyUsers(users) {
     // Remove users that are bots or appeared more than once
     const uniqueUsers = users.reduce((accumulator, user) => {
-      const { htmlUrl, login: username, type } = user;
+      const { htmlUrl, login: username, type, url } = user;
 
       const isUser = (type === 'User');
       const isNotDuplicate = !accumulator.has(username);
 
       if (isUser && isNotDuplicate) {
         accumulator.set(username, {
           handle: `@${username}`,
-          profileLink: htmlUrl
+          profileLink: htmlUrl,
+          url
```

Comment:
**suggestion (non-blocking):**

Without looking at the tests, I think it's a bit hard to understand the purpose of `url`. Maybe we rename it to `apiLink` to indicate that it's meant to be used in a request somehow?

If so, line 62 will need to be updated as well.

```suggestion
          apiLink: url
```

Reply:
On the other hand, I feel, we should keep it in sync with what the GitHub API sends in response? That way, it is easier to make changes in the future. I'd rather move this to ember-data models probably.